gradle-baseline-java/build.gradle

```diff
@@ -128,6 +128,10 @@ pluginBundle {
     }
 }
 
+tasks.named('processResources').configure {
+    duplicatesStrategy 'include'
+}
```

Comment:
I imagine this is due to gradle 7 failing on duplicate plugin resource files

Reply:
Yes. It fails for the plugin resources because they are defined [here](https://github.com/palantir/gradle-baseline/blob/2351209dc8f15158a9f145a166abce1c3ad6ae22/gradle-baseline-java/build.gradle#L97) and [here](https://github.com/palantir/gradle-baseline/tree/2351209dc8f15158a9f145a166abce1c3ad6ae22/gradle-baseline-java/src/main/resources/META-INF/gradle-plugins).

Let me add a comment.